test/Conversion/XTenNNToTorch/conv_transpose_2d.mlir

```diff
@@ -0,0 +1,230 @@
+// RUN: aten-opt --convert-xtennn-to-torch  -split-input-file %s | FileCheck %s
+
+func.func @with_output_shape(%arg0: tensor<1x480x36x22xf32>, %arg1: tensor<480x360x4x4xf32>, %arg2: tensor<360xf32>) -> tensor<1x360x72x44xf32> {
+  %0 = xten_nn.ConvTranspose %arg0, %arg1, %arg2 {dilation = array<i64: 1, 1>, group = 1 : i64, output_padding = array<i64: 0, 0>, pad = [array<i64: 1, 1>, array<i64: 1, 1>], stride = array<i64: 2, 2>} : (tensor<1x480x36x22xf32>, tensor<480x360x4x4xf32>, tensor<360xf32>) -> tensor<1x360x72x44xf32>
```

Comment:
Could you add a test where `group > 1` for completeness?

Reply:
Done